Library/Homebrew/software_spec.rb

```diff
@@ -423,6 +423,11 @@ def tab_attributes
     github_packages_manifest_resource_tab(github_packages_manifest_resource)
   end
 
+  sig { returns(Filename) }
+  def filename
+    Filename.create(resource.owner, @tag, @spec.rebuild)
```

Comment:
Seems weird no tests cover this, is this used anywhere right now?

Reply:
The tests currently use a double for it, so I think that's what it doesn't get coverage. Want me to add some separate coverage for it?

Reply:
Done with https://github.com/Homebrew/brew/pull/17049/commits/1607d04ad2e5b278b42cbcc61b8b1bf45a541475 -- I went with a separate test since the more invasive double was going to require a lot of nesting that would make the other tests harder to read, but I can shoehorn it in if that's your preference 🙂

Reply:
(I also had to move the `BottleSpecification` specs to their own file, since RuboCop was complaining about having two `RSpec` stanzas in `bottle_spec.rb`.)